### ISSUE TITLE
Add auto "Export data" on change

### ISSUE THREAD

**Issue by czarnyckm** [albertvaka/bettercounter#76]
An option to enable automatic backups after any change.

**Comment by MikeStirner** [comment]
Yeah as a setting perhaps? But maybe it could also save a daily snapshot at midnight local time if there has been any changes in the past 24 hours. But I like the backup after each change because that would make it easy to integrate the data into other apps in real-time.

**Comment by MikeStirner** [comment]
How easy is that, @albertvaka ?

**Comment by albertvaka** [comment]
It should be easy. I want to test how slow it is to do in on each increase (probably it's fast enough), but for sure we can at least have it once a day. And given we are going to have a settings page, it can be added there.

**Comment by MikeStirner** [comment]
This will be so sweet! Good to hear it should be easy!

**Comment by albertvaka** [comment]
Hi! I just pushed a new update (might take some days to get published) that includes a setting to auto-export the entries on each change. I assume this should be performant enough (it is on my phone with a decent amount of entries), but if not I can change it to be done once per day instead. Give it a try and let me know how it goes! :D
